lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -166,8 +164,50 @@ def transform_service_pay(service_pay_source)
         service_pay_target
       end
 
+      def transform_toxic_exposure(toxic_exposure_source)
+        toxic_exposure_target = Requests::ToxicExposure.new
+
+        gulf_war1990 = toxic_exposure_source['gulfWar1990']
+        gulf_war2001 = toxic_exposure_source['gulfWar2001']
+        if gulf_war1990.present? || gulf_war2001.present?
+          toxic_exposure_target.gulf_war_hazard_service =
+            transform_gulf_war(gulf_war1990, gulf_war2001)
+        end
+
+        toxic_exposure_target
+      end
+
       private
 
+      def transform_gulf_war(gulf_war1990, gulf_war2001)
+        gulf_war1990_value = gulf_war1990&.values&.any?(&:present?) && !none_of_these(gulf_war1990)
+        gulf_war2001_value = gulf_war2001&.values&.any?(&:present?) && !none_of_these(gulf_war2001)
```

Comment:
Is it possible for `gulf_war1990` or `gulf_war2001` to be nil, or are you simply accounting for that scenario should it occur?

Reply:
Most likely it would not happen organically, but we just wanted to be extra careful.